.gitignore

```diff
@@ -33,6 +33,9 @@ npm-debug.log
 # this depending on your deployment strategy.
 /priv/static/
 
+# Don't commit the SSL keys. Generate them with the mix phx.gen.cert command
```

Comment:
Can you add this to the README? Then this will be perfect!

Reply:
Done :)